packages/flutter_tools/lib/src/globals.dart

```diff
@@ -204,6 +204,22 @@ void printStatus(
   );
 }
 
+
+/// Display the [message] inside a box.
+/// For example, this is the generated output:
```

Comment:
most trivialest of nits: missing blank line here (see comment above)

Reply:
done